src/Orleans/Runtime/DedicatedThreadPool.cs

```diff
@@ -0,0 +1,756 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="Helios.Concurrency.DedicatedThreadPool.cs" company="Akka.NET Project">
+//     Copyright (C) 2009-2016 Lightbend Inc. <http://www.lightbend.com>
+//     Copyright (C) 2013-2016 Akka.NET project <https://github.com/akkadotnet/akka.net>
+// </copyright>
+//-----------------------------------------------------------------------
+
+/*
+ * Copyright 2015 Roger Alsing, Aaron Stannard
+ * Helios.DedicatedThreadPool - https://github.com/helios-io/DedicatedThreadPool
```

Comment:
@jthelin the NuGet package just installs the .CS file. No binaries. We wanted to make it easy for consumers to mark the class as `internal` just in case multiple libraries in the same project depend on it.

As far as licensing goes - preserving the original copyright header on the file and including the license text somewhere in the project (